tests/test_quantity.py

```diff
@@ -803,6 +827,20 @@ def testing_properties():
     print(
         f"User unit: {x._unit.user_unit}, multiplier: {x._unit.si_factor}, reduced_si_unit: {x._unit.si_unit}, si_value: {x._si_value}"
     )
-
-    c = q.Quantity(1.0, "C")
-    print(float(c))
+    print("")
+
+    c = q.Quantity(1.0, "C^1")
```

Comment:
On a different topic, does pint use `"C"` for degrees Celcius?

Reply:
> On a different topic, does pint use `"C"` for degrees Celsius? And if so, how does it cover coulomb? Similar question regarding the latter for this code.

@seanpearsonuk `C++ units library` uses `coulomb` for coulomb.

![image](https://user-images.githubusercontent.com/106588300/202401955-c9463d78-536f-4a50-ab1a-6ad108044bdf.png)